_config.yml

```diff
@@ -64,6 +64,10 @@ poliastro_url: http://poliastro.readthedocs.io/
 poliastro_mailinglist_dev: https://groups.io/g/poliastro-dev
 poliastro_riot: https://riot.im/app/#/room/#poliastro:matrix.org
 
+plasmapy_url: http://plasmapy.org
+plasmapy_mailinglist: http://groups.google.com/group/plasmapy
+plasmapy_riot: https://riot.im/app/#/room/#pyastro:matrix.org
```

Comment:
this is wrong!

Reply:
Damnit! Too many different things going on with too many similar names.